payments-core/api/payments-core.api

```diff
@@ -4974,9 +4974,9 @@ public final class com/stripe/android/paymentsheet/injection/PaymentSheetViewMod
 public final class com/stripe/android/paymentsheet/injection/PaymentSheetViewModelModule_Companion_ProvideGooglePayRepositoryFactory : dagger/internal/Factory {
 	public fun <init> (Ljavax/inject/Provider;Ljavax/inject/Provider;)V
 	public static fun create (Ljavax/inject/Provider;Ljavax/inject/Provider;)Lcom/stripe/android/paymentsheet/injection/PaymentSheetViewModelModule_Companion_ProvideGooglePayRepositoryFactory;
-	public fun get ()Lcom/stripe/android/paymentsheet/GooglePayRepository;
+	public fun get ()Lcom/stripe/android/googlepaylauncher/GooglePayRepository;
```

Comment:
Since `com.stripe.android` will be the umbrella covering payments and other non-payments functions, could we move it to `com.stripe.android.payments.core.googlepaylauncher`? 
(or `com.stripe.android.payments.googlepaylauncher` if we plan to move googlepay to a standalone module later)

Reply:
If we have `com.stripe.android.paymentsheet` seems like we should also have `com.stripe.android. googlepaylauncher`